charts/public-service/README.md

```diff
@@ -125,14 +125,14 @@ $ helm upgrade --install \
 ### Set up static IP address
 k8s by default reserves different IP addresses after each restart. In order to have a static one, you need to manually create it.
```

Comment:
That's not quite correct. The ephemeral external IP lives as long as your loadbalancer is existing. When the loadbalancer is destroyed, the IP is also destroyed. Static external IP addresses are not bound to the lifetime of the loadbalancer itself.

Reply:
https://github.com/commercetools/k8s-charts/pull/16/commits/2b963b14d4fd0cdd609f8150330c1b9e93969d0b